packages/controls/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './common/data-type-key'
 
 export * from './text-input'
+export * from './text-area'
```

Comment:
nit: no newline at end of file, Prettier should be configured so that this is taken care of automatically.